runtime/onert/core/src/exec/MinMaxData.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "MinMaxData.h"
+
+#include <iostream>
+
+namespace onert
+{
+namespace exec
+{
+
+RawMinMaxDumper::RawMinMaxDumper(const std::string &filename) : _filename(filename) {}
+
+void RawMinMaxDumper::dump(const exec::IOMinMaxMap &input_minmax,
+                           const exec::OpMinMaxMap &op_minmax) const
+{
+  // Find file is already exist for modifying
+  auto file = std::fopen(_filename.c_str(), "rb+");
```

Comment:
```suggestion
  auto file = std::fopen(_filename.c_str(), "wb+");
```
Sn success, `file` seems to be modified in `else` block.

Reply:
Also, you may use `ab+` without explicit file existence check though you still need to check whether it is a new file or not before reading `runs` from file.

Reply:
When I try to open with "ab+" mode, it cannot overwrite run count.